mvg/mvg.py

```diff
@@ -517,6 +518,47 @@ def create_tabular_measurement(
             json=body,
         )
 
+    def list_timestamps(
+        self,
+        sid: str,
+        offset: int = None,
+        limit: int = None,
+        order: SortOrder = None,
+    ) -> Dict[str, Union[int, List[int]]]:
```

Comment:
It looks hard to match this type with the description under "Returns". Shouldnt' the return type be `list`?

Reply:
Yes, you are correct -we are only returning "items". However, shouldn't we return the full Dict, so we can implement pagination on the client (same with list_measurements)?